diesel_derives/src/sql_function.rs

```diff
@@ -453,7 +454,12 @@ impl ToTokens for StrictFnArg {
     }
 }
 
-fn is_sqlite_type(ty: &syn::Type) -> bool {
+fn is_sqlite_type<T: ToTokens>(ty: &T) -> bool {
+    let ty = quote!(#ty).to_string();
+    let ty_stripped = ty
+        .strip_prefix("Nullable < ")
+        .and_then(|t| t.strip_suffix(" >"))
+        .unwrap_or(ty.as_str());
```

Comment:
Aggregations can have a nullable return type. If there's a way to handle that case without string manipulation then plz let know.

Reply:
We could try to deconstruct a concrete syn type here. I think `syn::Type` or `syn::TypePath` should be the correct one here. This would also handle fully qualified types correctly.

Reply:
I have very limited experience with the `syn` crate. Therefore, I don't understand what you mean by "try to deconstruct a concrete syn type". Can you please elaborate?

Reply:
Thanks, done. Sorry, I was busy the last two weeks and came around working on this just now.